src/WebSocketHandler.js

```diff
@@ -23,6 +25,8 @@ class WS extends EventEmitter {
   constructor(){
     super();
 
+    this.WebSocket = _WebSocket;
```

Comment:
I think it's worth leaving a comment explaining why this is done/the problem we had.

Reply:
Fixed. Thanks!